src/plone/rest/service.py

```diff
@@ -1,10 +1,10 @@
 # -*- coding: utf-8 -*-
-from Products.Five import BrowserView
+import zope.publisher.browser
```

Comment:
Any particular reason why you're using `zope.publisher.browser.BrowserView` here?
`zope.publisher.browser.BrowserView` in a Zope2 context gets a messed up acquisition chain. `Products.Five.BrowserView` fixes that.